### PR TITLE
Changes to the recentering method

### PR DESCRIPTION
**Pull request by chunhuazhou**
## DESCRIPTION OF CHANGES: 
Changing the recentering method to use control DA analysis as the member background for recentering, then add ensemble perturbations to this base state; remove beta from the recentering namelist (not used).

## TESTS CONDUCTED: 
Tested in an older workflow on Jet.

## ISSUE: 
This PR address the issue at https://github.com/NOAA-EMC/rrfs-workflow/issues/148

## CONTRIBUTORS (optional): 
@hu5970

### CODE REVIEW

**Comment by chunhuazhou** [comment]
Hold this PR for possible changes to the recentering variable list since a new background is now used.
Also it should go with the updated rrfs_utl code

**Comment by chan-hoo** [comment]
@chunhuazhou, could you please open an issue for this PR? It will make us maintain the project easier.

**Comment by chunhuazhou** [comment]
@chan-hoo Thanks for the reminder! Just created an issue and linked it to the PR.

**Comment by chan-hoo** [comment]
Thanks! :)

**Comment by chunhuazhou** [comment]
Variable list change is done. The  rrfs_utl code has been updated.

File: jobs/JRRFS_RUN_PREPSTART
Comment (hu5970) [inline jobs/JRRFS_RUN_PREPSTART:76]: this is not defined in deterministic cycle.

File: scripts/exrrfs_run_prepstart.sh
Comment (hu5970) [inline scripts/exrrfs_run_prepstart.sh:395]: ctrl_bkpath is not set right in control run

File: scripts/exrrfs_run_prepstart.sh
Comment (hu5970) [inline scripts/exrrfs_run_prepstart.sh:406]: should we define a new cycle_subtype for ensemble DA spin-up run?

**Comment by hu5970** [comment]
@MatthewPyle-NOAA We are testing this one for v0.7.7 if possible. Please hold off the tag and this PR for testing results. I should be able to send you some results later today or tomorrow. Also, I will  make a PR to update model hash again for the latest PR for RRFS is in today.

**Comment by MatthewPyle-NOAA** [comment]
Thanks @hu5970 Will keep an eye out.

**Review by hu5970 — APPROVED** [review]
looks good

**Comment by hu5970** [comment]
@MatthewPyle-NOAA I have test this on Dogwood and fixed couple of bugs. The current code should be good to go. I will update the v0.7.7 after you merge the PR and start to test again. Thanks, Ming

**Review by MatthewPyle-NOAA — APPROVED** [review]
If it is good enough for @hu5970 , it is good enough for me.  Approving.

**Pull request merged**
